src/main/webapp/aboutus.html

```diff
@@ -21,7 +18,7 @@
     <title>About Us</title>
     <link rel="stylesheet" href="/css/main.css">
   </head>
-  </head>
```

Comment:
Did we pickup a bonus </head> at some point?

Reply:
We had an extra `</head` closing tag and no `<body>` opening tag,

Reply:
Whoops! Thanks for taking care of that